.gitignore

```diff
@@ -1,3 +1,3 @@
 CS438_Project
 .idea/
-client/client
+Client/Client
```

Comment:
since we're keeping the folder names all lowercase, could you rename Client folder name to client?

Reply:
Sure